imdown-mobile/events/event_detailed.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from "react";
+import { FlatList, Image, Linking, Text, TouchableOpacity, View } from "react-native";
+import { EventDetailedStyles } from "./event_detailed_styles";
+import moment from 'moment';
+import Divider  from '../components/divider/divider'
+
+const DOWN_EMOJI_HEIGHT = 82
+const DOWN_EMOJI_WIDTH = 85
+const EVENT_PIC_HEIGHT = 130
+const EVENT_PIC_WIDTH = 130
+const ROW_BUTTON_HEIGHT = 40
+const ROW_BUTTON_WIDTH = 40
+
+const EventDetailed = (props) => {
+const [event, setEvent] = useState(props.route.params.event)
+
+	const renderRSVPUser = ({ item }: { item: String }) => {
+    return (
+      <View>
+            <Text style={EventDetailedStyles.rsvp_user}>{item}</Text>
+      </View>
+    );
+	};
+	
+  return (
+    <View style={EventDetailedStyles.container}>
+				{/* Event title + pic + details box */}
+        <View style={EventDetailedStyles.event_pic_and_title_container}>
+            { event.image_url ?  
+                <View style={EventDetailedStyles.event_picture}>
+                    <Image source = {{uri: event.image_url}}
+                    style = {{ width: EVENT_PIC_WIDTH, height: EVENT_PIC_HEIGHT, borderRadius: 15 }}
+                    /> 
+                </View> : 
+                <View></View>
+            }
+            <View style={EventDetailedStyles.event_title_container}>
+                <Text style={EventDetailedStyles.event_title}>
+                    {event.name}
+                </Text>
+                <Text style={EventDetailedStyles.event_time}>
+                    {`🗓 ${moment(event.start_ms).format('llll').toLocaleString()}`}
+                </Text>
+                <Text onPress={ ()=> Linking.openURL(`https://${event.url}`) } style={EventDetailedStyles.event_url}>
```

Comment:
Should the https:// part be in the url itself

Reply:
I felt that displaying the `https://` in the app for all the links would be a little too verbose, so I decided to leave it out of the url. the figma does the same

Reply:
![image](https://user-images.githubusercontent.com/19477036/88470025-5f7dd400-ceac-11ea-8e07-1ec866d24d29.png)

Reply:
Oh yeah I see that makes sense. I guess my concern is like this assumes https though maybe users shouldn't be visiting just http websites anyway LOL

Reply:
yeah we can probably add protocol once we're polishing things up. for now, I think the current state should suffice